common/components/internal-header/template.njk

```diff
@@ -1,9 +1,9 @@
 <header class="app-header {{ params.classes if params.classes }}" role="banner" data-module="header"
         {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
-  <div class="app-header__container {{ params.containerClasses | default('app-width-container') }}">
+  <div class="app-header__container {{ params.containerClasses if params.containerClasses }}">
 
     <div class="app-header__logo">
-      <a href="{{ params.homepageUrl | default('/') }}" class="app-header__link app-header__link--homepage">
+      <a href="{{ AUTH_BASE_URL }}/auth" class="app-header__link app-header__organisation-name">
```

Comment:
I'm a bit conscious of having this variables within the component as now they are tied specifically to the app configuration and are no longer independent. 

I think a nice approach would be to update the value of `homepageUrl` to `AUTH_BASE_URL + "/auth"` in [base.njk](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/d3b9972af9a0c322de782586255d9a57c05adc06/common/templates/layouts/base.njk#L96) where we define the header. Then the value is tied specifically to our implementation of this component and when testing the component you don't need to worry about setting global nunjucks values (with I think might be causing some of the unit test failures).

Reply:
Replaced `AUTH_BASE_URL + "/auth"` with `organisationHomepageUrl`, homepageUrl is already in use for the BaSM homepage